modules/navier_stokes/src/fvkernels/INSFVTKEDSourceSink.C

```diff
@@ -112,11 +114,16 @@ INSFVTKEDSourceSink::computeQpResidual()
 
     const auto & face_info_vec = libmesh_map_find(_face_infos, _current_elem);
     const auto & distance_vec = libmesh_map_find(_dist, _current_elem);
+    mooseAssert(distance_vec.size(), "Should have found a distance vector");
+    mooseAssert(distance_vec.size() == face_info_vec.size(),
+                "Should be as many distance vectors as face info vectors");
 
```

Comment:
Why are these cehck needed at this point? It seems to be adding computational burden every time that the kernel is called

Reply:
asserts are removed from opt mode. It only affects devel builds, in a very minor way. They are a good practice